### PR TITLE
Create form submitting new haircut

### PR DESCRIPTION
**Pull request by carlosaicrag**
renamed the client_haircut_avg_time table to client_haircut_time. created form so that users create a new row in the client haircuts table but also create a new row in the clieent_haircut_avg_time table. Pushed the url to the main splash when the user creates a new haircut.

### CODE REVIEW

File: app/controllers/api/client_haircuts_controller.rb [hunk 1]
```diff
@@ -8,9 +8,17 @@ def queue
     def create
         @client_haircut = ClientHaircut.new(client_haircut_params)
         @client_haircut.client_id = current_client_user.id
+        client_haircut_avg_time = ClientHaircutTime.avg_time(current_client_user.id,params[:client_haircut][:haircut_id],
+                                            params[:client_haircut][:barber_id])
+        if client_haircut_avg_time.length == 0
+            @client_haircut_avg_time = ClientHaircutTime.new(client_haircut_params)
+        else
+            @client_haircut_avg_time = client_haircut_avg_time[0]
```
Comment (linakherchi) [inline app/controllers/api/client_haircuts_controller.rb]: I thought that the client_haircut_avg_time would be a string, why are we keying at 0?

File: db/seeds.rb [hunk 2]
```diff
@@ -8,13 +8,15 @@
...
-barber5 = User.create(email:"steph@gmail.com", password:"*mwFMYKvQeLNS7vT", fname:"Steph", lname:"Curry", image_url:"./steph_curry_barber.jpg", working: true)
-barber6 = User.create(email:"steph@gmail.com", password:"*mwFMYKvQeLNS7vT", fname:"Steph", lname:"Curry", image_url:"./steph_curry_barber.jpg")
+barber1 = User.create(email:"dame@gmail.com", password:"starwars", fname:"Damian", lname:"lillard", image_url:"./dame_dolla.jpg", working: true)
+barber2 = User.create(email:"james@gmail.com",password:"starwars",fname:"James", lname:"Harden", image_url:"./james_harden.jpg", working: true)
+barber3 = User.create(email:"kyrie@gmail.com", password:"starwars", fname:"Kyrie", lname:"Irving", image_url:"./kyrie_irving.jpg", working: true)
+barber4 = User.create(email:"lebron@gmail.com", password:"starwars", fname:"Lebron", lname:"James", image_url:"./lbj.jpg", working: true)
+barber5 = User.create(email:"steph@gmail.com", password:"starwars", fname:"Steph", lname:"Curry", image_url:"./steph_curry_barber.jpg", working: true)
+barber6 = User.create(email:"steph@gmail.com", password:"starwars", fname:"Steph", lname:"Curry", image_url:"./steph_curry_barber.jpg")
```
Comment (linakherchi) [inline db/seeds.rb]: thank you ! it is better for testing

File: app/controllers/api/client_haircuts_controller.rb (hunk 1 above)
Comment (carlosaicrag) [reply app/controllers/api/client_haircuts_controller.rb]: The function ClientHaircutTime.avg_time is doing an active record queue which returns a collection.  There should only ever be one or zero items when the method is called so thats why i'm indexing into the first position if there is something found

File: db/seeds.rb (hunk 2 above)
Comment (carlosaicrag) [reply db/seeds.rb]: Yeah, that long password was getting really annoying

**Pull request merged**